v2/azure-arm.yaml

```diff
@@ -936,17 +931,20 @@ objectModelConfiguration:
       FlexibleServers_FirewallRule:
         $exportAs: FlexibleServersFirewallRule
         $supportedFrom: v2.0.0-alpha.2
-    2021-12-01-preview:
+    2022-01-01:
```

Comment:
Do we want this breaking change?

Reply:
Hasn't been released yet, so safe. I only just merged support for it like a week ago.